.github/workflows/tests.yml

```diff
@@ -17,23 +17,24 @@ jobs:
       - name: Install Dafny
         uses: dafny-lang/setup-dafny-action@v1
         with:
-          dafny-version: "3.9.1"
+          dafny-version: "nightly-2023-02-15-567a5ba"
 
       - name: Install lit
         run: pip install lit OutputCheck
 
       - name: Set up JS dependencies
         run: npm install bignumber.js
 
-      - name: Verify Dafny Code
-        run: lit --time-tests -v --param 'dafny_params=/verificationLogger:trx /verificationLogger:csv' .
-
-      - name: Generate Report
-        run: find . -name '*.csv' -print0 | xargs -0 --verbose dafny-reportgenerator summarize-csv-results --max-duration-seconds 10
-
-      - uses: actions/upload-artifact@v2  # upload test results
-        if: success() || failure()        # run this step even if previous step failed
-        with:
-          name: verification-results
-          path: '**/TestResults/*.trx'
-          if-no-files-found: error
+      - name: Verify Legacy Code
+        run: lit --time-tests -v 
+        ##--param 'dafny_params=/verificationLogger:trx /verificationLogger:csv' .
```

Comment:
We shouldn't be disabling parts of the CI, even temporarily.

Reply:
It looks like, in particular, the missing `.` at the end is causing the tests to fail at the moment.